cfme/fixtures/appliance.py

```diff
@@ -166,6 +166,18 @@ def temp_appliances_unconfig_funcscope_rhevm(appliance, pytestconfig):
         yield appliances
 
 
+@pytest.fixture(scope="module")
+def temp_appliances_unconfig_modscope_rhevm(appliance, pytestconfig):
+    with sprout_appliances(
+            appliance,
+            config=pytestconfig,
+            count=2,
+            preconfigured=False,
+            provider_type='rhevm'
```

Comment:
multiregion only supported on rhev?

Reply:
it is rather not supported on rhos. I temporarily added this limitation and get rid of it later

Reply:
there is BZ that multi-region works incorrectly on rhos

Reply:
maybe a TODO to update this at a later date? Optional though.

Reply:
I see a TODO below, that covers this

Reply:
there is already such todo :) 
https://github.com/ManageIQ/integration_tests/pull/9191/files#diff-28c42b11967ad7998e2cb41a79fe2992R18

regarding this fixture. I just copied it from similar fixture with function scope :)